### PR TITLE
Update tests to check for database creation recovery

### PR DESCRIPTION
**Pull request by Lipt0nas**

### CODE REVIEW

File: test/scenarios.py [hunk 1]
```diff
@@ -4728,11 +4728,84 @@
...
+                            "result": 0,
+                            "app_version": "*",
+                            "prod": false
+                        }""",
+                        ]
+                    ),
+                ]
+            ),
+        },
+        dbpath="/tmp/no-permissions",
```
Comment (matszczygiel) [inline test/scenarios.py:4807]: This one is deleted on `dev` I guess. Use `/root`

File: test/scenarios.py [hunk 2]
```diff
@@ -4728,11 +4728,84 @@
...
         },
         dbpath="/tmp/db/26-corrupted.sqlite",
     ),
+    Scenario(
+        "scenario26-1",
+        "Test if the instance can recover when file is innacessible",
+        {
+            "ren": ActionList(
+                [
+                    action.NoEvent(),
```
Comment (matszczygiel) [inline test/scenarios.py:4768]: You're missing the `DropPrivileges` actions. The runners by default run as a root

**Comment by Lipt0nas** [comment]
I see that the tests are quite different here than what is currently in `dev`, maybe it would be best to merge this when `main` is merged into `dev`?

**Comment by matszczygiel** [comment]
You can wait, but I don't see a point. Let's use `/root/db.sqlite` as in dev and we will free ourselves from the hassle to remember to change or merge later

**Pull request closed without merging**